packages/itwin/property-grid/src/components/PropertyGrid.tsx

```diff
@@ -303,6 +294,24 @@ export const PropertyGrid = ({
           }
         }
 
+        // Do any overrides on default menu options
+        if(defaultContextMenuOptions?.size && defaultContextMenuOptions.size > 0 ) {
```

Comment:
after the `if` or `for` there should be a space (look for all applicable examples)